### PR TITLE
Continuous deployment configuration for cloud.gov

### PR DESCRIPTION
**Pull request by adborden**
This will automatically deploy the `master` branch to cloud.gov, assuming build and tests are successful from CI.

What's left
- [x] waiting on cloud.gov org creation
- [x] configure `CF_DEPLOY_USER` and `CF_DEPLOY_PASSWORD` secrets in CI
- [x] update cd config for any cloud.gov details that may have changed (org/space)

### CODE REVIEW

**Comment by adborden** [comment]
@pkarman ready for review when you get a chance

File: Makefile [hunk 1]
```diff
@@ -11,4 +11,8 @@ run:
 	cd prototypes/form-wizard && \
 	npm run serve
 
-.PHONY: build setup run
+test:
+	cd prototypes/form-wizard && \
+	echo ok
```
Comment (pkarman) [inline Makefile:16]: maybe this could be something more substantial, just to verify the build? like 
```
stat _site/assets/js/agencies.js && echo ok
```
?

**Review by pkarman — APPROVED** [review]
I like how you've done the auth stuff, to keep creds out of github. I haven't tried the deployment stuff myself, but assume that once this gets merged, circleci will attempt to run this on master.

**Comment by pkarman** [comment]
looks like this branch needs a re-base in order to merge via GH

**Pull request merged**